### ISSUE TITLE
clarify relationship between this repo and SCBI-ForestGEO-Data/tree_mortality

### ISSUE THREAD

**Issue by teixeirak** [SCBI-ForestGEO/SCBImortality#4]
@ValentineHerr , let's clarify what goes in which repo and remove any overlap. I can't currently think of any reason why we need to have the data in [SCBI-ForestGEO-Data/tree_mortality](https://github.com/SCBI-ForestGEO/SCBI-ForestGEO-Data/tree/master/tree_mortality), but we should keep the folder with a simple readme pointing to this repo. Does that make sense to you?

**Comment by ValentineHerr** [comment]
Yes, I was thinking the same thing. I'll move the rest of the files over here.

**Comment by ValentineHerr** [comment]
Ok, I copied everything over, except the folder R_input_files that I believe should be deleted anyways.

@teixeirak, do you want me to go ahead and delete everything in SCBI-ForestGEO-Data/tree_mortality and add a README.md with a link to this repo or do you prefer to do it yourself?

**Comment by teixeirak** [comment]
That would be great; thanks!

**Comment by ValentineHerr** [comment]
done, I think I can close this now.
